app/test_main.py

```diff
@@ -1,3 +1,26 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_add_zeros_when_age_less_then_15() -> None:
```

Comment:
The test name `test_should_add_zeros_when_age_less_then_15` is not very descriptive. Consider renaming it to something like `test_cat_and_dog_age_less_than_15` to better reflect the condition being tested.